src/pybind/mgr/progress/module.py

```diff
@@ -434,10 +447,10 @@ class Module(MgrModule):
             runtime=True
         ),
         Option(
-            'persist_interval',
+            'sleep_interval',
             default=5,
             type='secs',
-            desc='how frequently to persist completed events',
+            desc='how frequently to wake up and update events',
```

Comment:
nit, interval is not identical to frequency. the larger the former the smaller the latter. so the combination of the document and the name of variable is confusing.

Reply:
this change no longer exists

Reply:
although it is unrelated to this PR, i'd keep this comment unresolved. as "5" is the interval between each call to persist the events, from user's perspective, the higher "frequency" the more frequently we persist the events. so this description is not user friendly.